db/migrate/20230424043200_add_post_to_comments.rb

```diff
@@ -0,0 +1,6 @@
+class AddPostToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_column :comments, :post_id, :integer
+    add_index :comments, :post_id
+  end
```

Comment:
It looks like you missed the migration steps that start at 3:40 of https://teamtreehouse.com/library/active-record-associations-in-rails/variations-on-foreign-key-migrations

```suggestion
  def change
    add_column :comments, :post_id, :integer
    add_index :comments, :post_id
    t.references :post, null: false, foreign_key: true
  end
```